test/Microsoft.DotNet.ProjectModel.Tests/GivenThatIWantToFilesMatchedWithPatternsInJson.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.DotNet.ProjectModel;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using System.Linq;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.ProjectModel.Tests
+{
+    public class GivenThatIWantToFilesMatchedWithPatternsInJson
+    {
+        private const string ProjectName = "some project name";
+        private readonly string ProjectFilePath = AppContext.BaseDirectory;
+
+        [Fact]
+        public void PackInclude_is_empty_when_it_is_not_set_in_the_ProjectJson()
+        {
+            var json = new JObject();
+            var project = GetProject(json);
+
+            project.Files.PackInclude.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void It_sets_PackInclude_when_packInclude_is_set_in_the_ProjectJson()
+        {
+            const string somePackTarget = "some pack target";
+            const string somePackValue = "some pack value";
+
+            var json = new JObject();
+            var packIncludeJson = new JObject();
+            json.Add("packInclude", packIncludeJson);
+
+            packIncludeJson.Add(somePackTarget, somePackValue);
+
+            var project = GetProject(json);
+
+            var packInclude = project.Files.PackInclude.FirstOrDefault();
+
+            packInclude.Target.Should().Be(somePackTarget);
+            packInclude.SourceGlobs.Should().Contain(somePackValue);
+        }
+
+        [Fact]
+        public void It_parses_the_builtIns_successfully()
+        {
+            var json = new JObject();
+            json.Add("excludeBuiltIn", "*.fs");
+            json.Add("exclude", "bin/");
+            json.Add("contentBuiltIn", "*.jpg");
+            json.Add("compileBuiltIn", "*.cs");
+            json.Add("resourceBuiltIn", "*.resx");
+            json.Add("publishExclude", "*.pdb");
+            Action action = () => GetProject(json);
+
+            action.ShouldNotThrow();
```

Comment:
This test should pass for arbitrary keys & values, right? It's not clear this test adds value...